h2o-genmodel/src/main/java/hex/genmodel/attributes/ModelJsonReader.java

```diff
@@ -314,6 +314,17 @@ private static Object convertBasedOnJsonType(final JsonElement convertFrom, fina
                         }
                         convertTo = arrS;
                         break;
+                    case STRING_ARR_ARR:
+                        final String[][] arrSS = new String[array.size()][];
+                        for (int i = 0; i < array.size(); i++) {
+                            final JsonArray arr2 = array.get(i).getAsJsonArray();
+                            arrSS[i] = new String[arr2.size()];
+                            for (int j = 0; j < array.size(); j++) {
```

Comment:
Shouldn't the for loop condition be `j < arr2.size()`?

Reply:
missing unit test

Reply:
thx, test improved to check for expected output